### ISSUE TITLE
Privacy Notice link fails required AA contrast ratio

### ISSUE THREAD

**Issue by youriwims** [mozilla/donate-wagtail#313]
### Issue: [Text elements must have sufficient color contrast against the background](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=AxeChrome)

**Element path: n**/a

**Snippet:** `<a href="#">Privacy Notice</a>`
<img width="459" alt="Screen Shot 2019-09-30 at 4 04 06 PM" src="https://user-images.githubusercontent.com/36484608/65912293-f3ba0000-e39b-11e9-89ab-a702493f8743.png">

**How to fix:**
Fix any of the following:
- Change text `color`

====

More Info:

Comments: n/a
Resources: https://webaim.org/resources/contrastchecker/?fcolor=595CF3&bcolor=111111

**Comment by nicklee** [comment]
I can update this to the Mozilla light blue #b7b9fa

**Comment by youriwims** [comment]
Yep, I think that's a great idea. I believe that's what's live on the foundation.mozilla.org site right now. cc: @beccaklam

**Comment by beccaklam** [comment]
@nicklee Yes all links should be #b7b9fa. It's included in this ticket: #273 which is a more comprehensive ticket detailing UI fixes for the footer.

**Comment by nicklee** [comment]
Updated along side https://github.com/mozilla/donate-wagtail/issues/273
